js/bGal.js

```diff
@@ -1,38 +1,64 @@
-class Bgal{
-    /* mimics the functionality of B-Gal, assuming Mg is a cofactor
-    and reaction takes place at 30 degrees C. */
-    constructor(mut){
+// Author : Richard Charczenko
+// Edited By : Gina Philipose, Rena Ahn, Zachary Mullen
+/* JavaScript File : bGal.js
+     The original File was a Python file (bGal.py) hosted with Flask
+     Translation By : Gina Philipose, Zachary Mullen
+*/
+// Last Update : August 7th, 2024

```

Comment:
Probably don't need to have last updated in the comments, as we have version control software to handle this for us and will add an extra line to the diff nearly every commit. This applies to the other files as well